paddle_image.py

```diff
@@ -0,0 +1,131 @@
+__copyright__ = "Copyright (c) 2021 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+
+from typing import Optional, Iterable, List, Any
+
+import numpy as np
+from jina import DocumentArray, Executor, requests
+
+
+def _batch_generator(data: List[Any], batch_size: int):
+    for i in range(0, len(data), batch_size):
+        yield data[i:min(i + batch_size, len(data))]
+
+
+class ImagePaddlehubEncoder(Executor):
+    """
+    :class:`ImagePaddlehubEncoder` encodes ``Document`` content from a ndarray,
+    potentially B x (Channel x Height x Width) into a ndarray of `B x D`.
+
+    Internally, :class:`ImagePaddlehubEncoder` wraps the models from `paddlehub`.
+    https://github.com/PaddlePaddle/PaddleHub
+
+    :param model_name: the name of the model. Supported models include
+        ``xception71_imagenet``, ``xception65_imagenet``, ``xception41_imagenet``,
+        ``vgg19_imagenet``, ``vgg16_imagenet``, ``vgg13_imagenet``, ``vgg11_imagenet``,
+        ``shufflenet_v2_imagenet``,``se_resnext50_32x4d_imagenet``,
+        ``se_resnext101_32x4d_imagenet``,  ``resnext50_vd_64x4d_imagenet``,
+        ``resnext50_vd_32x4d_imagenet``, `resnext50_64x4d_imagenet``,
+        ``resnext50_32x4d_imagenet``, ``resnext152_vd_64x4d_imagenet``,
+        ``resnext152_64x4d_imagenet``, ``resnext152_32x4d_imagenet``,
+        ``resnext101_vd_64x4d_imagenet``, ``resnext101_vd_32x4d_imagenet``,
+        ``resnext101_32x8d_wsl``, ``resnext101_32x48d_wsl``, ``resnext101_32x32d_wsl``,
+        ``resnext101_32x16d_wsl``, ``resnet_v2_50_imagenet``, ``resnet_v2_34_imagenet``,
+        ``resnet_v2_18_imagenet``, ``resnet_v2_152_imagenet``, ``resnet_v2_101_imagenet``,
+        ``mobilenet_v2_imagenet``, ``inception_v4_imagenet``, ``googlenet_imagenet``,
+        ``efficientnetb7_imagenet``, ``efficientnetb6_imagenet``, ``efficientnetb5_imagenet``,
+        ``efficientnetb4_imagenet``, ``efficientnetb3_imagenet``, ``efficientnetb2_imagenet``,
+        ``efficientnetb1_imagenet``, ``efficientnetb0_imagenet``, ``dpn68_imagenet``,
+        ``dpn131_imagenet``, ``dpn107_imagenet``, ``densenet264_imagenet``,
+        ``densenet201_imagenet``, ``densenet169_imagenet``, ``densenet161_imagenet``,
+        ``densenet121_imagenet``, ``darknet53_imagenet``, ``alexnet_imagenet``,
+    :param pool_strategy: the pooling strategy. Default is `None`.
+    :param channel_axis: The axis of the color channel, default is -3
+    :param default_batch_size: size of each batch
+    :param default_traversal_path: traversal path of the Documents, (e.g. 'r', 'c')
+    :param on_gpu: set to True if using GPU
+    :param args:  Additional positional arguments
+    :param kwargs: Additional keyword arguments
+    """
+    def __init__(
+            self,
+            model_name: Optional[str] = None,
+            pool_strategy: Optional[str] = None,
+            channel_axis: int = -3,
```

Comment:
I think channel axis should not be here as for clip

Reply:
this is not clip. 
It should be ok to keep this maybe? I know we should do this in normalizer